hexrd/unitcell.py

```diff
@@ -1246,6 +1209,50 @@ def color_orientations(self,
 
         self.stiffness = C
 
+    def is_editable(self, lp_name):
+        """
+        @author Saransh Singh, Lawrence Livermore National Lab
+        @date 03/17/2021 SS 1.0 original
+        @details check if a certain field in the lattice parameter 
+        is editable. this depends on the space group number or the
+        lattice class
+        """
+        res = False
+        _lpnamelist = list(_lpname)
+        index = _lpnamelist.index(lp_name)
+        editable_fields = _rqpDict[self.latticeType][0]
+        if(index in editable_fields):
+            res = True
+        return res
```

Comment:
This could be simplified to just:
```python
return index in editable_fields
```

And that removes the `res` variable altogether.

Reply:
ok, got it.